Cisco/cisco-secure-firewall/_meta/smart-descriptions.json

```
@@ -56,5 +56,61 @@
         "type": "requested"
       }
     ]
+  },
+  {
+    "value": "Connection for user {user.name} group {user.domain} from {source.ip}: {action.name}",
```

```suggestion
    "value": "Connection for user {user.domain}/{user.name} from {source.ip}: {action.name}",
```

you could write the domains using Windows nomenclature